util/etc/vscode-tasks.json

```diff
@@ -8,50 +8,50 @@
             "problemMatcher": [],
             "group": {
                 "kind": "build",
-                "isDefault": true,
-            },
+                "isDefault": true
+            }
         },
         {
             "label": "cargo dev fmt",
             "type": "shell",
             "command": "cargo dev fmt",
             "problemMatcher": [],
-            "group": "none",
+            "group": "none"
         },
         {
             "label": "cargo uitest",
             "type": "shell",
             "command": "cargo uitest",
             "options": {
                 "env": {
-                    "RUST_BACKTRACE": "1",
+                    "RUST_BACKTRACE": "1"
                     // This task will usually execute all UI tests inside `tests/ui` you can
                     // optionally uncomment the line below and only run a specific test.
                     //
                     // See: https://github.com/rust-lang/rust-clippy/blob/master/book/src/development/adding_lints.md#testing
                     //
                     // "TESTNAME": "<TODO>",
```

Comment:
Could you swap the order of these options? Otherwise, the format would be invalid, if `TESTNAME` would be uncommented :)